src/sage/topology/simplicial_complex.py

```diff
@@ -2002,6 +2116,30 @@ def suspension(self, n=1, is_mutable=True):
                                  rename_vertices=True)
         return self.suspension(1, is_mutable).suspension(int(n-1), is_mutable)
 
+    def union(self, right, is_mutable=True):
+        """
+        The union of this simplicial complex with another one. The set of
+        vertices of the union is the union of vertices of this complex and the
```

Comment:
```suggestion
        The union of ``self`` with ``right``.
        
        The set of vertices of the union is the union of vertices of ``self`` and the
```
However, I would change `right` to `other`.